src/cursor/cur_backup.c

```diff
@@ -202,6 +202,7 @@ __curbackup_close(WT_CURSOR *cursor)
 err:
 
     if (F_ISSET(cb, WT_CURBACKUP_FORCE_STOP)) {
+        /* TODO: Not sure if a cached checkpoint list invalidation is required here. */
```

Comment:
@sueloverso , I am wondering if it is okay to let the cached checkpoint list be as such here. Do we need to get rid if the cached checkpoint list? I do not know anything about the checkpoint taken for forced stop incremental backup.

Reply:
This is only a checkpoint of the metadata, not the full database. So all other table's lists should be valid. Only the metadata list would become invalid.

Reply:
That is what my thought was, so safe not to do anything here. I will remove the comment.